netket/nn/masked_linear.py

```diff
@@ -63,7 +63,7 @@ def __call__(self, inputs: Array) -> Array:
         Applies a masked linear transformation to the inputs.
```

Comment:
Applies `a->the` linear transformation

Reply:
The docstrings in `linear.py` indeed use 'a'